src/components/CharList/CharList.jsx

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+
+export default function CharList({ characters }) {
+  return (
+    <>
```

Comment:
you can minimize `div` soup by not using fragments when you're returning just one parent element